WordPress/src/main/java/org/wordpress/android/ui/reader/views/compose/horizontalpostlist/HorizontalPostListItemLoading.kt

```diff
@@ -0,0 +1,133 @@
+package org.wordpress.android.ui.reader.views.compose.horizontalpostlist
+
+import android.content.res.Configuration
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxHeight
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.lazy.LazyRow
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import org.wordpress.android.ui.compose.components.shimmer.ShimmerBox
+import org.wordpress.android.ui.compose.theme.AppColor
+import org.wordpress.android.ui.compose.theme.AppTheme
+import org.wordpress.android.ui.compose.unit.Margin
+
+@Composable
+fun HorizontalPostListItemLoading() {
+    val loadingColor = AppColor.Black.copy(
+        alpha = 0.08F
+    )
+    Column(
+        modifier = Modifier
+            .width(240.dp)
+            .height(340.dp),
+    ) {
+        Row(
+            modifier = Modifier.fillMaxWidth(),
+            verticalAlignment = Alignment.CenterVertically,
+        ) {
+            ShimmerBox(
+                modifier = Modifier
+                    .width(99.dp)
+                    .height(8.dp)
+                    .clip(shape = RoundedCornerShape(16.dp))
+                    .background(loadingColor),
+                isLoadingCompleted = false,
+            )
+        }
+        ShimmerBox(
```

Comment:
Maybe nitpick - I see creation of ShimmerBox instances with similar modifiers. Is there a way to further abstract these calls?

Reply:
@daniloercoli I think the color and shape repeats in many cases. The shape is not always the same, though, so maybe I can move the color to the `ShimmerBox` component? WDYT?

Reply:
👍 Feel free to merge this PR.